src/images/svg/begambleaware.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we start to put svg to their respective folder, so this will be in src/images/svg/responsible/begambleaware.svg, same goes for png